presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry.py

```diff
@@ -70,18 +71,55 @@ class RecognizerRegistry:
     def __init__(
         self,
         recognizers: Optional[Iterable[EntityRecognizer]] = None,
-        global_regex_flags: Optional[int] = re.DOTALL | re.MULTILINE |
-            re.IGNORECASE,
+        global_regex_flags: Optional[int] = re.DOTALL | re.MULTILINE | re.IGNORECASE,
+        supported_languages: Optional[List[str]] = None,
     ):
         if recognizers:
             self.recognizers = recognizers
         else:
             self.recognizers = []
         self.global_regex_flags = global_regex_flags
+        self.supported_languages = (
+            supported_languages if supported_languages else ["en"]
+        )
+
+    def _create_nlp_recognizer(
```

Comment:
nit: move privates below publics, also in analyzer_engine_provider.py

Reply:
@RoeyBC let's handle on the next PR